pkg/virtual/framework/fixedgvs/doc.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright 2021 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package fixedgvs provides the types (and underlying implementation)
+// required to build viertul workspaces which allow adding well-defined APIs
```

Comment:
virtual

Reply:
Fixed in rebased commit 994272b295369abe1e74e4c1709fca10fdeaab7b